frontend/app/index.tsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { ThemeProvider } from '@mui/material';
 
 import Home from './components/Home';
 import theme from './theme';
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+      retryOnMount: false,
+      staleTime: Infinity
+    },
+    mutations: { retry: false }
+  }
```

Comment:
`react-query` includes a lot of retry and refetch behavior by default. We may want some of this eventually, but for now I thought it seemed unnecessary and easier for testing to disable it all. See https://react-query.tanstack.com/guides/important-defaults